Source/RapyutaSimulationPlugins/Private/Core/RRROS2GameMode.cpp

```diff
@@ -11,12 +11,25 @@
 
 // RapyutaSimulationPlugins
 #include "Core/RRNetworkGameMode.h"
+#include "Core/RRTypeUtils.h"
+#include "Robots/Turtlebot3/TurtlebotBurger.h"
+#include "Robots/Turtlebot3/TurtlebotBurgerVehicle.h"
 #include "Tools/RRGhostPlayerPawn.h"
 #include "Tools/RRROS2ClockPublisher.h"
 
 ARRROS2GameMode::ARRROS2GameMode()
 {
     DefaultPawnClass = ARRGhostPlayerPawn::StaticClass();
+
+    // Static spawnable classes
```

Comment:
should avoid adding thses static classes in this base class for test.

Reply:
So basically, this is the place configuring spawnable entity classes, which is not only for test, but more to accommodate `SpawnEntity` srv. Otherwise, they have to be manually configured in all maps' Level Blueprint.

Reply:
move to .ini